### ISSUE TITLE
Adding new entity type to NER model 

### ISSUE THREAD

**Issue by ereday** [explosion/spaCy#5786]
## How to reproduce the behaviour

I went through the tutorial on adding an 'ANIMAL' entity to spaCy NER [here](https://github.com/explosion/spaCy/blob/master/examples/training/train_new_entity_type.py).

Hi, I am following the tutorial for adding new entity types to a NER system. Unlike in the tutorial, I have more than 6 sentences. The new type that I want to include to the model is 'Actor'. Basically, I went trough the sentences and annotated the occurrences of every Actor. Moreover, I also run the spaCy NER model on this additional dataset and include all of the NER model's entity predictions as additional data as well. (As long as the other predicted entities does not intersect with the 'Actor' entity span). I did this as it is recommended in the tutorial to fight against cat. forgetting problem. The final TRAIN_DATA array includes around 15K sentences and around 2100 of them includes 'Actor' entity. 

Then, in the code I added this new label by running `(ner.add_label('Actor'))` and since I am working with german data , I use `de_core_news_sm` model. Apart from these, I did not change anything at all. I use `python train_new_entity_type.py -m de_core_news_sm -o ./temp_output_dir/` to run the code. 

Unfortunately, I am getting some very bad loss values. It starts with 300k and even after 30 iterations it doesn't get any better. First, I thought that this might be because of the minibatch size, or  dropout. So I changed them as below:

```
#sizes = compounding(1.0, 4.0, 1.001) # This was the original one
sizes = compounding(20.0, 40.0, 1.001)        

#nlp.update(texts, annotations, sgd=optimizer, drop=0.35, losses=losses) # This was the original one 
 nlp.update(texts, annotations, sgd=optimizer, drop=0.1, losses=losses)
```

After these, I got the following loss values:

Losses {'ner': 33736.8532705307}
Losses {'ner': 31403.649552345276}
Losses {'ner': 30391.675664424896}
Losses {'ner': 29675.089672088623}
Losses {'ner': 29246.349004745483}
Losses {'ner': 29408.051209926605}
Losses {'ner': 28373.11535358429}
Losses {'ner': 28573.68544101715}
Losses {'ner': 28270.46219444275}
Losses {'ner': 27858.765252113342}
Losses {'ner': 27897.640738487244}
Losses {'ner': 27423.208258867264}
Losses {'ner': 27283.701818466187}
Losses {'ner': 27206.71978187561}
Losses {'ner': 27401.276213407516}
Losses {'ner': 26990.815222263336}
Losses {'ner': 26611.618522644043}
Losses {'ner': 26930.69779729843}
Losses {'ner': 26473.3650431633}
Losses {'ner': 26207.596734523773}
Losses {'ner': 26427.201930999756}
Losses {'ner': 26427.42885160446}
Losses {'ner': 26283.91295146942}
Losses {'ner': 26293.458068609238}
Losses {'ner': 26178.145215511322}
Losses {'ner': 26033.81904220581}
Losses {'ner': 26106.474103450775}

These are still very very high numbers and I am nowhere near enough to train the model. Am I doing something wrong here? I have already checked my annotations and data and everything seems fine. 

EDIT: Here is a couple of examples from the dataset I provide:

```
('"Es muss eine andere Lösung gefunden werden" , sagt Flüchtlingsrat-Geschäftsführerin Naujoks.', {'entities': [(52, 92, 'ACTOR')]})

('Die Stadt Schwerte will Asylsuchende in einem Gebäude einer ehemaligen Außenstelle des KZ Buchenwald unterbringen.\n', {'entities': [(87, 100, 'LOC'), (4, 18, 'ACTOR')]})
```

* **spaCy version:** 2.3.2
* **Platform:** Linux-5.6.16-300.fc32.x86_64-x86_64-with-fedora-32-Thirty_Two
* **Python version:** 3.6.5

**Comment by adrianeboyd** [comment]
1) I think the main problem is that you're trying to update a model with conflicting entity types. The provided German model `de_core_news_sm` has the entity types `PER` and `ORG` (and maybe `LOC`) that could all be `ACTOR` in some cases, so the model may end up pretty confused between its existing `PER` entities and the new `ACTOR` entities. The boundaries are also different, since `PER` would only include the name `Naujoks`. Instead of `de_core_news_sm`, you probably want to start with a blank German model (`spacy.blank("de")`).

2) I'm not that familiar with this annotation task, but I also think it's possible that `ACTOR` may not work out well as an entity type. What is the `ACTOR` in:

```
"Es muss eine andere Lösung gefunden werden" , sagt er Flüchtlingsrat-Geschäftsführerin Naujoks."
```

If `Flüchtlingsrat-Geschäftsführerin Naujoks` is not an actor in this sentence, then I think the default spacy NER model is probably not going perform well. You might use an NER model to identify potential actor-like entities and then filter those entities to ones that are more likely to be actors (using some kind of semantic role labeling?), but the spacy NER model on its own is probably not going to do a particularly good job making this kind of distinction. It wouldn't hurt to try it out with your data (I might be wrong!), but I think it's going to struggle to identify entity+agent or entity+agent-with-claim in comparison to just identifying named entities like person names.

And if you have to identify things like `er` above as an actor even though it's not a named entity, then it gets even harder for a model like the NER model. But I'm not sure (despite looking through several papers) what the detailed annotation guidelines for `actor` might look like.

**Comment by ereday** [comment]
Thank you for your answer and comments! I am actually trying something pretty similar to what you just described. My aim is to use a NER system to identify potential actor-like entities. I thought that it would be better to fine-tune the NER model by adding this new entity type (i.e. 'Actor') before using it to identify candidates.

**Comment by adrianeboyd** [comment]
I don't understand all the details about this annotation task, but I would have thought that `er` is the only actor in my modified example above?

In general, whether it makes sense to use the NER model depends a lot on how much of this task you're trying to solve with it. Without a lot of testing to check its performance, I wouldn't recommend it for anything beyond identifying named entities. If you collapse PER/LOC/ORG for your model that shouldn't be a problem, but other distinctions like agent vs. non-agent are probably going to be problematic.

**Comment by ereday** [comment]
Oh, my bad, I didn't realize that you modified the example! You're correct, `er` is the actor in that case. Thanks again, for your comments.
